content/intro-to-writing-data.md

```diff
@@ -233,14 +240,13 @@ Next, let’s add the instruction to the transaction we created. Then, call `sen
 
 ```typescript
 const transaction = new web3.Transaction()
-
 const programId = new web3.PublicKey(PING_PROGRAM_ADDRESS)
-const pingProgramDataId = new web3.PublicKey(PING_PROGRAM_DATA_ADDRESS)
+const programData = new web3.PublicKey(PING_PROGRAM_DATA_ADDRESS)
```

Comment:
This is a 'program ID' or 'program address' in Solana terminology. I don't think `programData` is a good name as this isn't data. Rather it's the address for a data account.

Reply:
updated to programDataId.